src/liquid/components/ld-input/ld-input.css

```diff
@@ -420,9 +420,13 @@
   }
 }
 
-:host([disabled]),
+.ld-input:has(input) {
+  transform: rotate(45deg);
+}
```

Comment:
What's happening here?

Reply:
Just testing if you're checking carefully enough 😆